actions/produce.py

```diff
@@ -42,4 +42,4 @@ def run(self, topic, message, hosts=None):
         result = producer.send_messages(topic, kafka_bytestring(message))
 
         if result[0]:
-            return result[0].__dict__
+            return result[0]._asdict()
```

Comment:
Just to be sure, is `result[0]` of `NamedTuple` type to have `_asdict()` method?

Reply:
@armab From this [answer](https://stackoverflow.com/questions/2166818/how-to-check-if-an-object-is-an-instance-of-a-namedtuple) it is a proper task to check this.

If you want to catch the error and not let it proliferate then we can do:

```
try:
    result[0]._asdict()
except AttributeError:
   ...
```